src/assets/scripts/client/ui/TutorialView.js

```diff
@@ -534,6 +498,16 @@ export default class TutorialView {
             position: tutorial_position
         });
 
+        this.tutorial_step({
+            title: 'Some Tips',
```

Comment:
We could use this section to link useful resources, such as the command reference, airport guides, etc.

Reply:
Commands are already linked at the end, and airport guides are available just above the Tutorial toggle. I'll add "check out the airport guide tab" tho in that section.